CMakeLists.txt

```diff
@@ -108,9 +108,9 @@ install_basic_package_files(${PROJECT_NAME} VARS_PREFIX ${VARS_PREFIX}
 
 include(AddUninstallTarget)
 
-if(ENABLE_TESTS)
+if(YARPWHOLEBODYINTERFACE_ENABLE_TESTS)
 #enable testing
-include(CTest)
+#include(CTest)
```

Comment:
no real reason, @francesco-romano  and I were trying to cleanup the CMakeLists before committing. If its needed, I wont cry ;)